I received this error after pressing the green play button: `error NU1104: Unable to find project 'C:\Users\anstewar\Documents\projects\streamstats\newfolder\NSSServices\Shared\Shared.csproj'. Check that the project reference is valid and that the project file exists.` ([info on error NU1104](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1104))

I checked GageStatsServices.csproj and see this: 
```
  <ItemGroup>
    <ProjectReference Include="..\..\NSSServices\Shared\Shared.csproj" />
    <ProjectReference Include="..\GageStatsAgent\GageStatsAgent.csproj" />
    <ProjectReference Include="..\GageStatsDB\GageStatsDB.csproj" />
  </ItemGroup>
```

The issue is that the NSSServices folder is not located in the project. I was able to resolve the error by following the [NSSServices README instructions](https://github.com/USGS-WiM/NSSServices#readme) first. I will update the GageStatsServices README so that users install NSSServices first.

--- a/README.md
+++ b/README.md
@@ -22,6 +22,8 @@ optional paramters
 
 [.NET Core](https://www.microsoft.com/net/core#windowscmd)
 
+[NSSServices](https://github.com/USGS-WiM/NSSServices)
+
 ## Getting Started
 
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
@@ -29,6 +31,8 @@ These instructions will get you a copy of the project up and running on your loc
 ### Installing
 
 https://help.github.com/articles/cloning-a-repository/
+
+Clone the GageStatsServices in the same directory as NSSServices. 
 
 Open the solution file (.sln) using preferred IDE. Visual Studio Community is recommended.
 

--- a/README.md
+++ b/README.md
@@ -1,4 +1,6 @@
 ![WiM](wimlogo.png)  
+# GageStats Services
+REST web services used in communication with the StreamStats Gage Statistics DB.
 
 ### Prerequisites
 

--- a/README.md
+++ b/README.md
@@ -1,20 +1,4 @@
-![WiM](wimlogo.png)
-
-# GageStats Services
-This project was created using the WIM services gagestats. The WIM Services gagestats usage is outlined below:
-## Installation of gagestats
-[WIM.Services.GageStats NuGet package](https://www.nuget.org/packages/WIM.Services.GageStats/):`dotnet new -i WIM.Services.GageStats`
-
-### GageStats Usage
-`mkdir newfolder`
-
-`cd newfolder\`
-
-`dotnet new wimservices`
-
-optional paramters
- `-E true` - enables service database options
-  
+![WiM](wimlogo.png)  
 
 ### Prerequisites
 

